content/doc/book/installing/_docker.adoc

```diff
@@ -52,15 +52,15 @@ Due to the use of a privileged container, this is recommended, though it require
 This environment variable controls the root directory where Docker TLS certificates are managed.
 <8> Maps the `/certs/client` directory inside the container to a Docker volume named `jenkins-docker-certs` as created above.
 <9> Maps the `/var/jenkins_home` directory inside the container to the Docker volume named `jenkins-data`.
-This will allow for other Docker containers controlled by this Docker container's Docker daemon to mount data from Jenkins.
+This allows for other Docker containers controlled by this Docker container's Docker daemon to mount data from Jenkins.
```

Comment:
Phrasing seems odd to me with the "for" in the phrase.  However, my English grammar is weak, so I am perfectly fine if you correct me and tell me that it is more correct to say "allow for other" than to say "allows other".  Optional.
```suggestion
This allows other Docker containers controlled by this Docker container's Docker daemon to mount data from Jenkins.
```

Reply:
My English grammar is super poor, but I've got a few tools to help.
One of them got rid of the _for_. 🤷